<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Interface</class>
 <widget class="QWidget" name="Interface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>479</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Interface</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="do_but">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>621</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 4);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Выполнить!</string>
   </property>
  </widget>
  <widget class="QComboBox" name="eprint_box">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Классический</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Канонический</string>
    </property>
   </item>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>704</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLineEdit" name="result_line">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="4">
       <widget class="QPushButton" name="change_len_but">
        <property name="text">
         <string>Изменить размер </string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="enter_pol_but">
        <property name="text">
         <string>Ввод полинома</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="change_an_but">
        <property name="text">
         <string>Изменение a_n</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="enter_X_but">
        <property name="text">
         <string>Ввод Х</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="change_print_but">
        <property name="text">
         <string>Вывод полинома</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QPushButton" name="index_change_buttom">
        <property name="font">
         <font>
          <pointsize>7</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Изменение корня по индексу</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>60</y>
     <width>255</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="1">
     <widget class="QLineEdit" name="new_size_edit"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="size_label">
      <property name="text">
       <string>Введите новый размер =</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="index_change_label">
      <property name="text">
       <string>Введите индекс корня =</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="index_change_edit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>101</width>
     <height>55</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="1" column="1">
     <widget class="QLabel" name="polin_num_label">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="polin_text_label">
      <property name="text">
       <string>Корень №</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="an_label_2">
      <property name="text">
       <string>Введите an</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>237</width>
     <height>94</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <widget class="QLabel" name="x_label">
      <property name="text">
       <string>X = </string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLineEdit" name="im_edit"/>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="i_label">
      <property name="text">
       <string>+i</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="an_label">
      <property name="text">
       <string>a = </string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="re_edit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="exit_buttom">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>0</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
